rep-2000.rst

```diff
@@ -208,13 +208,19 @@ Crystal Clemmys (December 2018 - December 2019)
 
 Targeted platforms:
 
-+--------------+-----------------------+------------------------------+------------------------+---------------------------+------------------------+
-| Architecture | Ubuntu Bionic (18.04) |     MacOS Sierra (10.12)     |   Windows 10 (VS2017)  | Ubuntu Xenial (16.04)     | Debian Stretch (9)     |
-+==============+=======================+==============================+========================+===========================+========================+
-|    amd64     |  Tier 1 [d][a][s]     |           Tier 1 [a][s]      |          Tier 1 [a][s] |           Tier 2 [s]      |        Tier 3  [s]     |
-+--------------+-----------------------+------------------------------+------------------------+---------------------------+------------------------+
-|    arm64     |  Tier 1 [d][a][s]     |                              |                        |           Tier 2  [s]     |        Tier 3 [s]      |
-+--------------+-----------------------+------------------------------+------------------------+---------------------------+------------------------+
++--------------+-----------------------+----------------------+---------------------+-----------------------+--------------------+---------------+
+| Architecture | Ubuntu Bionic (18.04) | MacOS Sierra (10.12) | Windows 10 (VS2017) | Ubuntu Xenial (16.04) | Debian Stretch (9) | OpenEmbedded  |
+|              |                       |                      |                     |                       |                    | Thud (2.6) /  |
+|              |                       |                      |                     |                       |                    | webOS OSE [s] |
++==============+=======================+======================+=====================+=======================+====================+===============+
+|    amd64     |  Tier 1 [d][a][s]     |    Tier 1 [a][s]     |     Tier 1 [a][s]   |       Tier 2 [s]      |     Tier 3 [s]     |               |
++--------------+-----------------------+----------------------+---------------------+-----------------------+--------------------+---------------+
+|    arm64     |  Tier 1 [d][a][s]     |                      |                     |       Tier 2  [s]     |     Tier 3 [s]     |               |
++--------------+-----------------------+----------------------+---------------------+-----------------------+--------------------+---------------+
+|    arm32     |                       |                      |                     |                       |                    |   Tier 3 [s]  |
++--------------+-----------------------+----------------------+---------------------+-----------------------+--------------------+---------------+
+|     x86      |                       |                      |                     |                       |                    |   Tier 3 [s]  |
```

Comment:
Is this significantly different that amd64? I think that this could be collapsed into the amd64 row. If we're adding a row it would be i386, but we've generally stopped targeting that platform as there's almost no CPUs with that architecture being produced these days. OpenEmbedded docs can document that it can work for i386 but I'd rather not just expand our support matrix as documented here.

Reply:
Agree, will remove the row.

We added it because we support [qemu images](https://www.yoctoproject.org/docs/1.8/yocto-project-qs/yocto-project-qs.html#setting-up-the-environment-and-starting-the-qemu-emulator) and packages based on both i386 (qemux86) and amd64 (qemux86-64). It can be listed in OpenEmbedded docs instead as you mentioned.